pkg/constants/constants.go

```diff
@@ -31,4 +31,13 @@ const (
 
 	BPF_LOG_ERR   = 0
 	BPF_LOG_DEBUG = 3
+
+	// Ip(0.0.0.2 | ::2) used for control command, e.g. KmeshControl | ByPass
+	ControlCommandIp4 = "0.0.0.2"
```

Comment:
In the IPv6 environment, control commands such as management and bypass commands triggered by 0.0.0.1 return 101 (Network is unreachable), and sockops:BPF_SOCK_OPS_TCP_CONNECT_CB hook cannot be triggered. ::1 indicates that control commands can be triggered, but the obtained local_ip value is also ::1. The actual pod IP address cannot be obtained.
Solution:
0.0.0.2/::2 is used as the IP address of the control command.